afb-binding/src/verbs.rs

```diff
@@ -240,7 +258,8 @@ pub(crate) fn register_verbs(apiv4: AfbApiV4,api: &mut AfbApi, config: BindingCf
         config.engy_api,
         config.ocpp_api,
         msg_evt,
-        config.basic_charging_enabled
+        config.basic_charging_enabled,
```

Comment:
Why this change ?

Reply:
Probably I was testing something by removing it. Then when I put it back I added a comma without a specific purpose.